server/graphql/queries.js

```diff
@@ -5,12 +5,43 @@ import {
 } from 'graphql';
 
 import {
+  CollectiveType,
+  UserType,
   EventType
 } from './types';
 
 import models from '../models';
 
 const queries = {
+  Collective: {
+    type: CollectiveType,
+    args: {
+      collectiveSlug: {
+        type: new GraphQLNonNull(GraphQLString)
+      }
+    },
+    resolve(_, args) {
+      return models.Group.findOne({
+        where: { slug: args.collectiveSlug.toLowerCase() }
+      })
+    }
+  },
+  /*
+   * Given a collective slug, returns all users
+   */
+  allUsers: {
```

Comment:
is this needed? Shouldn't there just be a single graphql query that can get the group and all its users?

```query collective {
    collective(slug: $groupSlug) {
        id,
        name,
        ...,
        users {
            id, 
            name,
            role,
            ...
        }
    }
}